Widgets 2.0 (#4028)
DataDog/documentation: content/en/graphing/widgets/top_list.md

---
title: Top List Widget
kind: documentation
further_reading:
- link: "graphing/dashboards/timeboard/"
  tag: "Documentation"
  text: "Timeboards"
- link: "graphing/dashboards/screenboard/"
  tag: "Documentation"
  text: "Screenboard"
---

> The top list visualization is perfect when you want to see the list of hosts with the most or least of any metric value, such as highest consumers of CPU, hosts with the least disk space, etc: 

Reviewer: ```suggestion
The top list visualization enables you to see the list of hosts with the most or least of any metric value, such as highest consumers of CPU, hosts with the least disk space, etc: 
```

{{< img src="graphing/widgets/toplist/toplist.png" alt="Top List" responsive="true">}}

## Setup

{{< img src="graphing/widgets/toplist/toplist_setup.png" alt="Top List" responsive="true" style="width:80%;">}}

### Configuration

1. Choose the data to graph:
    * Metric: See [the main graphing documentation][1] to configure a metric query.
    * APM Events: See [the trace search documentation][2] to configure an APM event query.
    * Log Events: See [the log search documentation][3] to configure an APM event query.

> 2. Optional - Configure a conditional formating depending of your entries values.

Reviewer: ```suggestion
2. Optional: configure conditional formatting depending of your entries' values.
```

### Options
#### Global time

> On Screenboard only, choose whether or not your widget has a custom timeframe or the global timeframe of the Screenboard.

Reviewer: ```suggestion
On screenboards only, choose whether your widget has a custom timeframe or the screenboard's global timeframe.
```
